packages/world/src/utils/EnergyUtils.sol

```diff
@@ -26,29 +27,23 @@ using ObjectTypeLib for ObjectTypeId;
 
 function getLatestEnergyData(EntityId entityId) view returns (EnergyData memory, uint128, uint128) {
   EnergyData memory energyData = Energy._get(entityId);
-
-  // Calculate how much time has passed since last update
   uint128 timeSinceLastUpdate = uint128(block.timestamp) - energyData.lastUpdatedTime;
+
   if (timeSinceLastUpdate == 0) {
     return (energyData, 0, 0);
   }
 
-  // Update timestamp for all cases
   energyData.lastUpdatedTime = uint128(block.timestamp);
 
   if (energyData.energy == 0) {
     return (energyData, 0, timeSinceLastUpdate);
   }
 
-  // Calculate energy drain
   uint128 energyDrained = timeSinceLastUpdate * energyData.drainRate;
-
   uint128 depletedTime = 0;
-  // Update accumulated depleted time if it ran out of energy on this update
+
   if (energyDrained >= energyData.energy) {
-    // Calculate when it ran out by determining how much time it took to drain the energy
```

Comment:
why were these comments removed?

Reply:
was trying to clean up some of the comments as they were kind of redundant, but yeah this one can stay